reefer_monitoring/v1/reefer_v1.0.0-Beta-1.yaml

```diff
@@ -0,0 +1,524 @@
+          example: 6
+        cargoProbe4Temperature:
+          type: number
+          x-stoplight:
+            id: 3dkqi3m46ix4l
+          format: float
+        currentPhaseA:
+          $ref: '#/components/schemas/currentPhaseA'
+        currentPhaseB:
+          $ref: '#/components/schemas/currentPhaseB'
+        currentPhaseC:
+          $ref: '#/components/schemas/currentPhaseC'
+        dischargePressure:
+          $ref: '#/components/schemas/dischargePressure'
+        dischargeTemperature:
+          $ref: '#/components/schemas/dischargeTemperature'
+        evaporatorTemperature:
+          $ref: '#/components/schemas/evaporatorTemperature'
+        lineFrequency:
+          $ref: '#/components/schemas/lineFrequency'
+        lineVoltage:
+          $ref: '#/components/schemas/lineVoltage'
+        returnProbe1Temperature:
+          $ref: '#/components/schemas/returnProbe1Temperature'
+        returnProbe2Temperature:
+          $ref: '#/components/schemas/returnProbe2Temperature'
+        returnTemperature:
+          $ref: '#/components/schemas/returnTemperature'
+        suctionPressure:
+          $ref: '#/components/schemas/suctionPressure'
+        suctionTemperature:
+          $ref: '#/components/schemas/suctionTemperature'
+        supplySensor1Temperature:
+          $ref: '#/components/schemas/supplySensor1Temperature'
+        supplySensor2Temperature:
+          $ref: '#/components/schemas/supplySensor2Temperature'
+        supplyTemperature:
+          $ref: '#/components/schemas/supplyTemperature'
+      title: Measurements
+    reeferResponse:
+      type: object
+      x-stoplight:
+        id: 1101e866d0266
+      properties:
+        alarms:
+          $ref: '#/components/schemas/alarms'
+        controller:
+          $ref: '#/components/schemas/controller'
+        equipmentReference:
+          $ref: '#/components/schemas/equipmentReference'
+        equipmentOptions:
+          $ref: '#/components/schemas/equipmentOptions'
+        geoLocation:
+          $ref: '#/components/schemas/geoLocation'
+        isConnectedToPowerSource:
+          type: boolean
+          description: 'Indicates if the source is connected to power, aligned to COA UDM P2'
+        lastDefrost:
+          $ref: '#/components/schemas/lastDefrost'
+        measurements:
+          $ref: '#/components/schemas/measurements'
+        setpoints:
+          $ref: '#/components/schemas/setpoints'
+        reeferSettings:
+          $ref: '#/components/schemas/reeferSettings'
+        unitModel:
+          $ref: '#/components/schemas/unitModel'
+    reeferSettings:
+      type: object
+      x-stoplight:
+        id: 89583c1e5a249
+      properties:
+        controlledAtmosphereMode:
+          $ref: '#/components/schemas/controlledAtmosphereMode'
+        defrosting:
+          $ref: '#/components/schemas/defrosting'
+        efficiencySetting:
+          $ref: '#/components/schemas/efficiencySetting'
+    setpoints:
```

Comment:
I know it is not my expertise - but isn't `airExchange` (+ unit) missing in the setPoints?

Reply:
We have asked for the API specifications to consider further set points and what can be influenced, as I had a similar question on the humidity its in set points but not in the measurements looking at the COA UDM.  So good to point out.

In COA this is called ventilation and is only a measurement.  So we decided to go with as is and have added to a list of questions in an ira ticket for COA members and SME's.